### DISCUSSION TITLE
How to enable "Run | Debug" lens?

### DISCUSSION THREAD

**Discussion by 1rodri** [mrcjkb/rustaceanvim#419] [Q&A] [answered]
There is a GIF in the Usage/Features section of the README.md that shows the "Run | Debug" lens above the main function:

<b>Structural search replace</b>

  - Searches the entire buffer in normal mode.
  - Searches the selection in visual mode.
  
  ```vim
  :RustLsp ssr {query}
  ```
  ```lua
  vim.cmd.RustLsp { 'ssr', '<query>' --[[ optional ]] }
  ```

  ![tty](https://github.com/mrcjkb/rustaceanvim/assets/12857160/b61fbc56-ab53-48e6-bfdd-eb8d4de28795)

I haven't been able to get this working and wanted to know if there's a setting I'm missing somewhere. The rust-analyzer docs say it should be enabled by default when CodeLens is enabled (`rust-analyzer.lens.run.enable`) so I'm not sure what the problem is. Is it still possible to enable this?

**Comment by mrcjkb** [comment, marked as answer]
Hey 👋 

See `:h vim.lsp.codelens.refresh` 😃

**Reply by 1rodri** [reply]
Ahhh yep I forgot to set up the BufEnter/InsertLeave autocmd. Thanks!

**Comment by devinlittle** [comment]
Quick question how do you get the  compilation status on the bottom right of your screen

**Reply by mrcjkb** [reply]
https://github.com/j-hui/fidget.nvim
